Client-Side/adoptForm/adoptForm.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>AdoptADog</title>
+    <link rel="stylesheet" href="../navbar/navbar.css">
+    <link rel="stylesheet" href="adoptForm.css">
+</head>
+<body>
+    <script type="text/javascript" src="../navbar/navbar.js"></script>
+
+    <script>loadNavbar();</script>
+
+    <main class="mainBody">
+        <form action="" class="adoptionForm">
+            <header>Take the first step to take your<br>fluffy friend home!</header>
+            <a class="sliderLink" href="">What is required of me?</a><br><br>
+            <label for="inputFirstName">FIRST NAME *</label><br>
+            <input type="text" name="inputFirstName" placeholder="Enter your first name"><br><br>
```

Comment:
<br> shouldn't be used to space things outside of text. Rather use "margin-bottom: "